### PR TITLE
[tests-only] adjust expected-failure list and archive related tests

### PR DESCRIPTION
**Pull request by saw-jan**
## Related Issue
- https://github.com/owncloud/ocis/issues/4636
- https://github.com/owncloud/ocis/issues/7208

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/copySpaces.feature [hunk 1]
```diff
@@ -706,10 +706,12 @@ Feature: copy file
...
     When user "Alice" copies file "/textfile1.txt" from space "Personal" to "/BRIAN-Folder" inside space "Shares" using the WebDAV API
-    Then the HTTP status code should be "204"
-    And for user "Alice" the content of the file "/BRIAN-Folder" of the space "Shares" should be "ownCloud test text file 1"
-    And as "Alice" file "/textfile1.txt" should exist
-    And user "Alice" should not have any received shares
+    Then the HTTP status code should be "400"
+    And as "Alice" folder "Shares/BRIAN-Folder/sample-folder" should exist
+    And as "Brian" folder "BRIAN-Folder/sample-folder" should exist
+    But as "Alice" file "Shares/BRIAN-Folder" should not exist
+    And as "Alice" file "Shares/textfile1.txt" should not exist
```
Comment (S-Panta) [inline tests/acceptance/features/apiSpacesShares/copySpaces.feature:713]: Why would this exist if the file is copied inside the BRIAN-Folder in Share drive?

Comment (saw-jan) [reply tests/acceptance/features/apiSpacesShares/copySpaces.feature:713]: didn't get the point.

Comment (saw-jan) [reply tests/acceptance/features/apiSpacesShares/copySpaces.feature:713]: the COPY is not done inside BRIAN-Folder folder but to BRIAN-Folder

**Review by ScharfViktor — COMMENTED** [review]
https://github.com/owncloud/web/issues/11080
should we use zip for all users-agents in tests?

**Comment by saw-jan** [comment]
> [owncloud/web#11080](https://github.com/owncloud/web/issues/11080) should we use zip for all users-agents in tests?

But we still can get tar archive using `output-format=tar` query param. `zip` is the default one. IMO, let's test that `output-format=tar` works atleast via API

**Comment by ScharfViktor** [comment]
>  IMO, let's test that `output-format=tar` works atleast via API. 

yeah, let leave it 
But since we are using zip, I would check in the api that zip works for different user-agents.
```
Examples:
      | user-agent | archive-type |
      | Linux      | zip          |
      | Windows NT | tar          |
| Windows NT | zip        |
```

**Comment by saw-jan** [comment]
>But since we are using zip, I would check in the api that zip works for different user-agents.

Yeah, I tried to mix-match the formats:
https://github.com/owncloud/ocis/blob/29b9f9b07de15b9f8abc0b221b208ca96032a503/tests/acceptance/features/apiArchiver/downloadById.feature#L24-L26
https://github.com/owncloud/ocis/blob/29b9f9b07de15b9f8abc0b221b208ca96032a503/tests/acceptance/features/apiArchiver/downloadById.feature#L42-L44

Will it be enough?

**Comment by ScharfViktor** [comment]
> Will it be enough?
yeah

**Pull request merged**